async-graphql-actix-web/src/lib.rs

```diff
@@ -16,21 +18,21 @@ pub use subscription::WSSubscription;
 
 /// Extractor for GraphQL request
 ///
-/// It's a wrapper of `QueryBuilder`, you can use `GQLRequest::into_inner` unwrap it to `QueryBuilder`.
-/// `async_graphql::IntoQueryBuilderOpts` allows to configure extraction process.
-pub struct GQLRequest(QueryBuilder);
+/// It's a wrapper of `GqlQueryBuilder`, you can use `GQLRequest::into_inner` unwrap it to `GqlQueryBuilder`.
+/// `async_graphql::IntoGqlQueryBuilderOpts` allows to configure extraction process.
+pub struct GQLRequest(GqlQueryBuilder);
```

Comment:
Should `GQLRequest` be updated too?

Reply:
yes I think we should change `GQLResponse` and `GQLRequest'.
There also another `GQLError` in http module, but it's just a wrapper around `&GqlError` to implement serialize, I guess we can implement `Serialize` on `GqlError` directly now